NEWS.md

```diff
@@ -1,3 +1,28 @@
+# simtrial 0.4.0
+
+## New features
+
+- RMST test is available (#188, thanks @LittleBeannie).
+- Milestone test is available (#199, #204, #211, #237, thanks @LittleBeannie).
+- Test version of `sim_gs_n()` is available with coming unit testing (#195, #201, #208, #212, thanks @LittleBeannie and @jdblischak).
+- Users have the flexibility to create custom interim and final analyses cuttings according to their specific requirements by utilizing the `create_cutting_test()` function. (#215, #221, thanks @jdblischak).
```

Comment:
`create_cutting_test()` was renamed to `create_test()` in the follow-up PR. In other words, we don't need two separate bullet points for `create_test()`